script/bootstrap

```diff
@@ -0,0 +1,266 @@
+then
+    sudo apt-get -y install \
+                    git \
+                    cmake \
+                    build-essential \
+                    python-dev \
+                    libgl1-mesa-glx \
+                    freeglut3-dev \
+                    libopenal-dev \
+                    libsdl-gfx1.2-dev \
+                    libvorbis-dev \
+                    libjpeg-dev \
+                    libpng-dev \
+                    libgtk2.0-dev \
+                    libgtk-3-dev \
+                    libboost-python-dev \
+                    libboost-log-dev \
+                    libboost-regex-dev
+elif [ $LINUX_ID == "opensuse-leap" ]
+then
+    sudo zypper install -y  libboost_log1_66_0-devel \
+                            libboost_python-py2_7-1_66_0-devel \
+                            libboost_python-py3-1_66_0-devel \
+                            libboost_system1_66_0-devel \
+                            libboost_filesystem1_66_0-devel \
+                            libboost_thread1_66_0-devel \
+                            libboost_regex1_66_0-devel \
+                            libboost_chrono1_66_0-devel \
+                            libboost_atomic1_66_0-devel \
+                            cmake \
+                            gcc-c++ \
+                            freeglut-devel \
+                            libopenal0 \
+                            openal-soft-devel \
+                            libSDL-1_2-0 \
+                            libSDL-devel \
+                            libvorbis-devel \
+                            libjpeg-turbo \
+                            libjpeg62-devel \
+                            libpng16-devel \
+                            expat \
+                            libexpat-devel \
+                            libgtk-2_0-0 \
+                            gtk2-devel \
+                            libgtk-3-0 \
+                            gtk3-devel \
+                            python-devel \
+                            python3-devel \
+                            git \
+                            rpm-build
+elif [ $LINUX_ID == "fedora" ] && [ $LINUX_VERSION_ID -eq 32 ]
+then
+    sudo dnf install -y git \
+                        cmake \
+                        boost-devel \
+                        boost-python3-devel \
+                        freeglut-devel \
+                        gcc-c++ \
+                        openal-soft-devel \
+                        SDL-devel \
+                        libvorbis-devel \
+                        libjpeg-turbo-devel \
+                        libpng-devel \
+                        expat-devel \
+                        gtk2-devel \
+                        gtk3-devel \
+                        python3-devel \
+                        rpm-build \
+                        make
+elif [ $LINUX_ID == "fedora" ] && ([ $LINUX_VERSION_ID -eq 31 || $LINUX_VERSION_ID -eq 30])
```

Comment:
could probably do a `$LINUX_VERSION_ID -ge 30 && $LINUX_VERSION_ID -lt 32` so we can do inclusive comparisons

Reply:
Done